src/Symfony/Component/Debug/Tests/phpt/fatal_with_nested_handlers.phpt

```diff
@@ -24,7 +24,9 @@ var_dump(array(
 $eHandler[0]->setExceptionHandler('print_r');
 
 if (true) {
-    class Broken implements \Serializable {};
+    class Broken implements \Serializable
```

Comment:
This is unnecessarily verbose. Can we integrate into fixer exception for such simple construct?

Reply:
Yes ... but it's only two more lines. I guess it's acceptable.

Reply:
I agree. Let's try to be as "compliant" as possible.